packages/docs-app/src/getting-started.md

```diff
@@ -110,31 +110,27 @@ For more information, see [Understanding TypeScript](#blueprint/reading-the-docs
 @## Vanilla JS APIs
 
 JS components are built using React, but that does not limit their usage to only React applications.
-You can render any component in any JavaScript application with `ReactDOM.render`. Think of it like
+You can render any component in any JavaScript application with `render`. Think of it like
```

Comment:
These docs generally seem a bit unnecessary to have included for Blueprint. Perhaps we should replace this entire section with links to the core React docs?

Reply:
Yeah agreed with that